app/views/shared/_order_detail_cell.html.haml

```diff
@@ -6,6 +6,10 @@
     - if show_reservation && od.reservation
       %br
       %em= od.reservation
+
+    - if od.product.offline?
```

Comment:
I commented in the wrong file; the offline warning sign I meant is in `app/views/shared/_reservations_table.html.haml`.